### DISCUSSION TITLE
Unable to connect remotely to Security Onion Server

### DISCUSSION THREAD

**Discussion by larstchet** [Security-Onion-Solutions/securityonion#7241] [Unsupported Versions]
I found that anytime I reboot the server I am unable to access the server via web interface. I installed that SO on Ubuntu 20.04. When I log into my Ubuntu account and run the following command I find most of the services to show "Wait-Start". Is this normal? Can I have the services start automatically after a reboot. Sorry, I'm new to Linux and SO. 

Thanks, 
Chet

**Comment by tomjones1977** [comment]
How long are you waiting for the services to start?  There are a lot of services that start in sequence, it can take up to 10 mins for them to all start properly.

**Comment by larstchet** [comment]
Perhaps I was too impatient. They appear to be up now. Is there a way to see the startup mode services are in. Is it similar to Windows modes of Automatic, Automatic Delayed, Manual...etc?

**Reply by larstchet** [reply]
Spoke to soon. I had access this morning but now it seems like I no longer have access? It appears I have to be logged into the ubuntu console before it will allow me to access the web interface from another computer? Can this be changed?

**Comment by dougburks** [comment]
> Is there a way to see the startup mode services are in.

Services should automatically start on boot. If they are taking a long time to start, then your machine may be underpowered.

> I had access this morning but now it seems like I no longer have access? It appears I have to be logged into the ubuntu console before it will allow me to access the web interface from another computer? Can this be changed?

Have you run `so-allow` to allow traffic through the firewall?
https://docs.securityonion.net/en/2.3/so-allow.html
